### PR TITLE
Added some adjectives and names

### PR DESCRIPTION
**Pull request by Morgy93**
Signed-off-by: Thomas Hauschild thomas@ulfertsprygoda.de

### CODE REVIEW

**Comment by cpuguy83** [comment]
```
These files are not properly gofmt'd:
 - pkg/namesgenerator/names-generator.go
```

Can you run `gofmt -s -w pkg/namesgenerator/names-generator.go`, and amend the current commit, then force push?

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -364,6 +373,9 @@ var (
 		// Srinivasa Ramanujan - Indian mathematician and autodidact who made extraordinary contributions to mathematical analysis, number theory, infinite series, and continued fractions. - https://en.wikipedia.org/wiki/Srinivasa_Ramanujan
 		"ramanujan",
 
+		// Sally Kristen Ride was an American physicist and astronaut. https://en.wikipedia.org/wiki/Sally_Ride
```
Comment (tiborvass) [inline pkg/namesgenerator/names-generator.go]: You can also add she was the first American woman in space, and the youngest American astronaut.

**Comment by Morgy93** [comment]
@cpuguy83 done.

**Comment by cpuguy83** [comment]
@Morgy93 Can you add the comments @tiborvass suggested?
Much respect for Sally.
I can't believe she wasn't already on the list!

**Comment by Morgy93** [comment]
@tiborvass @cpuguy83 There you go.

**Comment by cpuguy83** [comment]
LGTM

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**